src/dl_txt_pack.cpp

```diff
@@ -77,12 +77,7 @@ inline double dl_strtod(const char* str, char** endptr)
 
 inline float dl_strtof(const char* str, char** endptr)
 {
-#if defined(_MSC_VER) && _MSC_VER < 1800 // strtof was defined first in MSVC2013
```

Comment:
I'm pretty sure strtof was added in msvc not that long ago :(

Reply:
Oh, I should have commented on this change.

On line 17 in this file you have solved this exact issue, and I just saw that you fixed in here as well. Since it is enough to fix this issue in one way (the line 17 way) this could be removed and make the code less cluttered.

Reply:
:+1: